app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
+        super().__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self) -> None:
+        print(f"{self.nickname} is eating {self._favourite_dish}")
```

Comment:
The `eat_favourite_dish` method correctly prints the message using the `nickname` and `_favourite_dish` attributes. Ensure that the `nickname` attribute is inherited and accessible from the `Player` class.